notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p>This is kind of you to include but imho is not necessary. It is safe to assume if the user is running this they know their project ID, and if they don't, they probably need to do some beginner tutorials first</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>This contents was copied over from the template notebook. Brad and I will discuss about it.</p>